packages/lib/availability.ts

```diff
@@ -65,21 +65,15 @@ export function getWorkingHours(
   },
   availability: { days: number[]; startTime: ConfigType; endTime: ConfigType }[]
 ) {
-  // clearly bail when availability is not set, set everything available.
   if (!availability.length) {
-    return [
-      {
-        days: [0, 1, 2, 3, 4, 5, 6],
-        // shorthand for: dayjs().startOf("day").tz(timeZone).diff(dayjs.utc().startOf("day"), "minutes")
-        startTime: MINUTES_DAY_START,
-        endTime: MINUTES_DAY_END,
-      },
-    ];
+    return [];
```

Comment:
BTW, this fixes the unrelated bug that we default to ALL availability when no availability is set in a schedule.

Reply:
Had to update users fixtures since test users we're being created without availability.

Reply:
Oh! Good that we caught that as this is an impossible situation due to the onboarding flow these days 💯 - nice catch!